README.md

```diff
@@ -708,7 +708,7 @@ This is the first release of x16-rom by the X16Community team
 ### Release 35
 
 * API Fetures
-	* new KERNAL API: low-level and high-level 320x200@256c bitmap graphics
+	* new KERNAL API: low-level and high-level 320x240@256c bitmap graphics
```

Comment:
I'm very reluctant to change the content of old changelogs.

Reply:
i'll discard this change